ui/appui-react/src/appui-react/backstage/BackstageManager.tsx

```diff
@@ -90,6 +70,6 @@ export const useIsBackstageOpen = (manager: BackstageManager) => {
  * @public
  */
 export const useBackstageManager = () => {
-  const [manager] = React.useState(UiFramework.backstageManager);
+  const [manager] = React.useState(UiFramework.backstage);
```

Comment:
This is a breaking change as well (changed return type from `BackstageManager` to `FrameworkBackstage`?